.php-cs-fixer.dist.php

```diff
@@ -15,7 +15,7 @@
         ],
         'return_type_declaration' => [
             'space_before' => 'none'
-        ],
+        ]
```

Comment:
Nit: revert unneeded change
```suggestion
        ],
```